pkg/authorization/plugin.go

```diff
@@ -102,7 +110,7 @@ func (a *authorizationPlugin) initPlugin() error {
 				plugin, e = plugins.Get(a.name, AuthZApiImplements)
 			}
 			if e != nil {
-				err = e
+				err = ErrInvalidPlugin
```

Comment:
The error is already logged in https://github.com/anusha-ragunathan/docker/blob/fae904af02a184833d2cd5ce9fdd61a4083707c7/pkg/authorization/middleware.go#L62